src/amo/components/UserProfile/index.js

```diff
@@ -103,7 +160,71 @@ export class UserProfileBase extends React.Component<Props> {
       return `/users/edit`;
     }
 
-    return `/user/${user.username}/edit/`;
+    return `${this.getURL()}edit/`;
+  }
+
+  getReviewsPage() {
+    const { location } = this.props;
+
+    const currentPage = parseInt(location.query.page, 10);
+
+    return Number.isNaN(currentPage) || currentPage < 1 ? 1 : currentPage;
+  }
+
+  renderReviews() {
+    const { location, i18n, isOwner, reviews, reviewCount } = this.props;
+
+    if (!isOwner || !reviews || reviews.length < 1) {
+      return null;
+    }
+
+    const paginator =
+      reviewCount && reviewCount > DEFAULT_API_PAGE_SIZE ? (
+        <Paginate
+          LinkComponent={Link}
+          count={reviewCount}
+          currentPage={this.getReviewsPage()}
+          pathname={this.getURL()}
+          queryParams={location.query}
+        />
+      ) : null;
+
+    return (
+      <CardList
+        className="UserProfile-reviews"
+        footer={paginator}
+        header={i18n.gettext('My reviews')}
+      >
+        <ul>
+          {reviews.map((review) => {
+            const reviewBodySanitized = sanitizeHTML(nl2br(review.body), [
+              'br',
+            ]);
+
+            return (
+              <li key={String(review.id)}>
+                <div className="AddonReviewListItem">
```

Comment:
This should move to a `UserReview` component and should include any styles that were defined in `AddonReviewListItem`.

Ideally, [`AddonReviewListItem`](https://github.com/mozilla/addons-frontend/blob/ca492751c7dd4482ecefdccd5c1af779953e358f/src/amo/components/AddonReviewListItem/index.js#L234) should be refactored to use `UserReview` but that could come in a separate issue since it won't be a small task.

Reply:
Yes, I was very sad not to be able to reuse the existing component :/

Reply:
Filed: https://github.com/mozilla/addons-frontend/issues/5416

Reply:
Why not just move it to `UserReview` since you're already adding code here anyway? The `AddonReviewListItem` changes can definitely come later.

Reply:
ah, you mean creating this new component here? The diff is `+740 −46` already.